netmap/netmap_contract.go

```diff
@@ -312,6 +312,25 @@ func UpdateState(state int, publicKey interop.PublicKey) {
 	}
 }
 
+// UpdateStateIR method tries to change node state in the network map.
+// Should be invoked in notary-enabled environment by the alphabet.
```

Comment:
add somehow info that this should be invoked "*only* in notary-enabled...", please

Reply:
Fixed.